docs/Config_Reference.md

```diff
@@ -1584,6 +1588,10 @@ control_pin:
 #pin_move_time: 0.680
 #   The amount of time (in seconds) to wait for the BLTouch pin to
 #   move up or down. The default is 0.680 seconds.
+#   This determines if Klipper should command the pin to move up
+#   between each probe attempt when performing a multiple probe
+#   sequence. Read the directions in docs/BLTouch.md before setting
+#   this to False. The default is True.
```

Comment:
This text change looks like a copy-and-paste error.

Reply:
I have kept the existing description of the parameter in the documentation. and renamed the parameter in the "probe" section to be logically separate.
I can rewert this back to same naming in both "probe" and "bltouch" sections

Reply:
Sorry, my mistake. Made a commit with fixes.